app/models/invite.rb

```diff
@@ -109,7 +113,7 @@ def domain_matches?(email)
 
   def can_be_redeemed_by?(user)
     return false if !self.redeemable?
-    return true if self.email.blank? && self.domain.blank?
```

Comment:
This is a good question -- I've replaced this because the previous meaning of this was "if it's an invite link that is not tied to an email or domain, then of course the user can redeem". With the fix in this PR that is not the case, if the user has already redeemed an invite at least once then they can never redeem it again. So that `InvitedUser` check trumps all.